paymentsheet/src/test/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodDefinitionTest.kt

```diff
@@ -19,10 +20,10 @@ internal class PaymentMethodDefinitionTest {
     fun `isSupported returns false for PaymentMethodDefinitions that do not meet add requirements`() {
         val metadata = PaymentMethodMetadataFactory.create(
             stripeIntent = SetupIntentFixtures.SI_REQUIRES_PAYMENT_METHOD.copy(
-                paymentMethodTypes = listOf("card", "klarna")
+                paymentMethodTypes = listOf("card", "grabpay")
             )
         )
-        assertThat(KlarnaDefinition.isSupported(metadata)).isFalse()
+        assertThat(GrabPayDefinition.isSupported(metadata)).isFalse()
```

Comment:
Not for this PR, but it would be good if we didn't have to update this test every time we add support or more support for the LPM used in the test. I believe this is tied down by `PaymentMethodRegistry` though correct?

Reply:
The alternative would be to create a "fake" PaymentMethodDefinition inline that is unsupported for setup.

In theory a lot of these tests have the same problem.